### PR TITLE
fix dependabot reported security issues

### PR DESCRIPTION
**Pull request by mhiaror**
*Issue #, if available:*
Dependabot security fixes

*Description of changes:*
fix dependabot reported security issues

- update package.json dependencies to resolve the security vulnerabilities
- fix webpack config to not override window console logger
- remove unused dependencies (storybook)
- downgrade `sass-loader` as latest version is not supported by webpack4

*Testing Details:*
- used https://github.com/amazon-connect/amazon-connect-chat-ui-examples/ to load the chat widget and test. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -142,26 +139,30 @@
...
     "postcss-loader": "3.0.0",
     "postcss-preset-env": "6.0.6",
     "postcss-safe-parser": "4.0.1",
-    "react-dev-utils": "^11.0.4",
+    "react-dev-utils": "^12.0.0",
     "react-render-html": "^0.6.0",
     "react-test-renderer": "^16.6.3",
     "resolve": "1.8.1",
-    "sass-loader": "^12.1.0",
+    "sass-loader": "^10.1.1",
```
Comment (swiszm-amazon) [inline package.json:152]: Why was downgrading needed here?

File: configuration/webpack.config.prod.js [hunk 2]
```diff
@@ -152,7 +152,11 @@ module.exports = {
             // Pending futher investigation:
             // https://github.com/terser-js/terser/issues/120
             inline: 2,
-            drop_console: true,
+            pure_funcs: [
```
Comment (swiszm-amazon) [inline configuration/webpack.config.prod.js:155]: Why do we need this change? There is a separate logger in this package that should be used instead anyways

File: package.json [hunk 3]
```diff
@@ -104,9 +104,6 @@
   "devDependencies": {
     "@babel/core": "^7.1.0",
     "@babel/preset-env": "^7.2.0",
-    "@storybook/addon-info": "4.0.11",
-    "@storybook/addons": "^4.0.11",
-    "@storybook/react": "^4.0.11",
```
Comment (swiszm-amazon) [inline package.json:109]: Are these storybook dependencies just for development and we weren't using them? Can you include details in PR overview

**Review by swiszm-amazon — COMMENTED** [review]
Left a few comments. Can you also add testing details in PR overview?

File: package.json [hunk 4]
```diff
@@ -5,9 +5,9 @@
   "dependencies": {
     "@babel/runtime": "7.9.2",
     "@emotion/core": "^10.0.35",
-    "@svgr/webpack": "^5.4.0",
+    "@svgr/webpack": "^6.2.1",
     "@types/jest": "^26.0.19",
-    "amazon-connect-chatjs": "^1.1.7",
+    "amazon-connect-chatjs": "^1.1.10",
```
[the remark this answers is not in the document]
Comment (mhiaror) [reply package.json:10]: done

File: package.json (hunk 1 above)
Comment (mhiaror) [reply package.json:152]: Currently the `npm i` in the package is broken bec `sass-loader:12.x` needs webpack 5 https://github.com/webpack-contrib/sass-loader/blob/v12.1.0/package.json#L44
if you checkout the repo locally, it's not building.

File: configuration/webpack.config.prod.js (hunk 2 above)
Comment (mhiaror) [reply configuration/webpack.config.prod.js:155]: I got this change from `non-prod` webpack config https://code.amazon.com/packages/AmazonConnectChatInterface/blobs/bcd1de70812fde41d1c47bdbbaeac19580fe0b82/--/configuration/webpack.config.prod.js#L157
Without this change - currently the application does not load when I tested my changes in https://github.com/amazon-connect/amazon-connect-chat-ui-examples/ -- Root Cause: console.info/console.log are overridden to `undefined` which breaks/errors during app rendering.

File: package.json (hunk 3 above)
Comment (mhiaror) [reply package.json:109]: Storybook is not used in the public repo; these dependencies are not used.

**Review by haomingli2020 — APPROVED** [review]

**Review by swiszm-amazon — APPROVED** [review]

**Pull request merged**